frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -209,8 +209,7 @@ def clear_old_logs(days=30):
 	@frappe.whitelist()
 	def retry_sending(self):
 		if self.status == "Error":
-			self.status = "Not Sent"
-			self.save(ignore_permissions=True)
+			frappe.db.set_value(self.doctype, self.name, "status", "Not Sent")
```

Comment:
this is a bit of a doozy.

if someone uses this in somewhere else like:

```py
def something():
    ...
    doc.retry_sending()
    ...
```

then we'd want to make use of `.save()` as the status would need to reflect in the doc.